external/PATCH-01-STRATUM

```diff
@@ -692,22 +707,23 @@ index f553054d..3f41e9e1 100644
 +
    }
  }
-
+ 
 diff --git a/stratum/hal/lib/common/common.proto b/stratum/hal/lib/common/common.proto
-index f7a72d70..45736661 100644
+index f7a72d70..cc65cddc 100644
 --- a/stratum/hal/lib/common/common.proto
 +++ b/stratum/hal/lib/common/common.proto
-@@ -243,6 +243,23 @@ enum FecMode {
+@@ -243,6 +243,24 @@ enum FecMode {
    FEC_MODE_AUTO = 3;
  }
-
+ 
 +// SW backend port Types
 +enum SWBackendPortType {
 +  PORT_TYPE_NONE = 0;
-+  PORT_TYPE_TAP = 1;
-+  PORT_TYPE_LINK = 2;
-+  PORT_TYPE_SOURCE = 3;
-+  PORT_TYPE_SINK = 4;
++  PORT_TYPE_VHOST = 1;
```

Comment:
Does the change from LINK to VHOST naming also means the code and documentation in IPDK need to be changed?

Reply:
That's not required. Internal logic introduced on line 284 in gnmi_cli.cc file will change the port type to vhost if device-type is virtual device and port type is link. This is to avoid confusion for existing users. Also, both vhost user ports and pci link ports are treated as link ports by DPDK backend.